content/automate/ManageIQ/Transformation/TransformationHosts/Common.class/__methods__/utils.rb

```diff
@@ -16,6 +19,19 @@ def self.get_runners_count_by_host(host, handle = $evm)
               handle.vmdb(:service_template_transformation_plan_task).where(:state => 'active').select { |task| task.get_option(:transformation_host_id) == host.id }.size
             end
 
+            def self.host_max_runners(host, factory_config, max_runners = DEFAULT_HOST_MAX_RUNNERS, handle = $evm)
```

Comment:
@fdupont-redhat Is this just a Utility class which only has class methods that you want to embed into other classes. We don't need to have an initialize or the call at the bottom if you are going to just call it from other methods.

Reply:
I noticed this as well as there is an empty `main` method defined.  I'm good with including those changes here if appropriate but that refactoring could be in a separate PR as well since it is unrelated.

Reply:
Yes, it's just a utility class. It is meant to be added as an embedded method. Should I remove it ? It's a bit out of scope for this PR. Maybe another cleanup PR ?

Reply:
Ok, so I removed the unneeded parts: initialize, main and call at end of file.